docs/schedule/example-feed.de.md

```diff
@@ -0,0 +1,178 @@
+
+## trips.txt
+
+    route_id,service_id,trip_id,trip_headsign,block_id
+    A,WE,AWE1,Downtown,1
+    A,WE,AWE2,Downtown,2
+
+## stop_times.txt
+
+    trip_id,arrival_time,departure_time,stop_id,stop_sequence,pickup_type,drop_off_type
+    AWE1,0:06:10,0:06:10,S1,1,0,0
+    AWE1,,,S2,2,1,3
+    AWE1,0:06:20,0:06:30,S3,3,0,0
+    AWE1,,,S5,4,0,0
+    AWE1,0:06:45,0:06:45,S6,5,0,0
+    AWD1,0:06:10,0:06:10,S1,1,0,0
+    AWD1,,,S2,2,0,0
+    AWD1,0:06:20,0:06:20,S3,3,0,0
+    AWD1,,,S4,4,0,0
+    AWD1,,,S5,5,0,0
+    AWD1,0:06:45,0:06:45,S6,6,0,0
+
+## calender.txt
+
+    service_id,monday,tuesday,wednesday,thursday,friday,saturday,sunday,start_date,end_date
+    WE,0,0,0,0,0,1,1,20060701,20060731
+    WD,1,1,1,1,1,0,0,20060701,20060731
+
+## calendar_dates.txt
+
+Dieses Beispiel zeigt Serviceausnahmen für den Unabhängigkeitstag im Jahr 2006. Am Montag, dem 3. Juli 2006, wird der reguläre Wochentagsverkehr`(service_id=WD`) unterbrochen`(exception_type=2`). Stattdessen verkehrt an diesem Tag ein Wochenenddienst`(service_id=WE`)`(exception_type=1`). Die gleiche Änderung gilt auch für Dienstag, den 4. Juli.
+
+    service_id,date,exception_type
+    WD,20060703,2
+    WE,20060703,1
+    WD,20060704,2
+    WE,20060704,1
+
+## fare_attributes.txt
+
+    fare_id,price,currency_type,payment_method,transfers,transfer_duration
+    1,0.00,USD,0,0,0
+    2,0.50,USD,0,0,0
+    3,1.50,USD,0,0,0
+    4,2.00,USD,0,0,0
+    5,2.50,USD,0,0,0
+
+## fare_rules.txt
+
+    fare_id,route_id,origin_id,destination_id,contains_id
+    a,TSW,1,1,
+    a,TSE,1,1,
+    a,GRT,1,1,
+    a,GRJ,1,1,
+    a,SVJ,1,1,
+    a,JSV,1,1,
+    a,GRT,2,4,
+    a,GRJ,4,2,
+    b,GRT,3,3,
+    c,GRT,,,6
+
+## shapes.txt
+
+    shape_id,shape_pt_lat,shape_pt_lon,shape_pt_sequence,shape_dist_traveled
+    A_shp,37.61956,-122.48161,1,0
+    A_shp,37.64430,-122.41070,2,6.8310
+    A_shp,37.65863,-122.30839,3,15.8765
+
+## frequencies.txt
+
+    trip_id,start_time,end_time,headway_secs
+    AWE1,05:30:00,06:30:00,300
+    AWE1,06:30:00,20:30:00,180
+    AWE1,20:30:00,28:00:00,420
+
+## transfers.txt
+
+    from_stop_id,to_stop_id,transfer_type,min_transfer_time
+    S6,S7,2,300
+    S7,S6,3,
+    S23,S7,1,
+
+## pathways.txt
+
+    pathway_id,from_stop_id,to_stop_id,pathway_mode,is_bidirectional
+    E1N1,E1,N1,2,1
+    E2N1,E2,N1,2,1
+    E3N1,E3,N1,2,1
+    N1-N2,N1,N2,6,1
+    N2-N1,N2,N1,7,1
+    N2N3,N2,N3,2,1
+    N2-B1,N2,B1,4,0
+    B1-N2,B1,N2,4,0
+    B1B2,B1,B2,2,1
+
+    B1B3,B1,B3,1,1
+    B2B4,B2,B4,1,1
+
+    E4N3,E4,N3,2,1
+    E5N3,E5,N3,2,1
+    N3-N4,N3,N4,6,1
+    N4-N3,N4,N3,7,1
+    N4B3,N4,B3,2,1
+    N4-B3,N4,B3,4,0
+    B3-N4,B3,N4,4,0
+    B3B4,B3,B4,2,1
+
+## levels.txt
+
+    level_id,level_index,level_name,elevation
+    L0,0,Street,0
+    L1,-1,Mezzanine,-6
+    L2,-2,Southbound,-18
+    L3,-3,Northbound,-24
+
+## translations.txt
+
+    table_name,field_name,language,translation,record_id
+    stops,stop_name,en,Tokyo Station,stopid000001
```

Comment:
Unless I didn't understand the note about the "files not being connected", this stop doesn't appear in `stops.txt` above.

Reply:
I believe each `.txt` example is different and unrelated to each other.